PowerControlPanel/Classes/StartUp/Resources/StringResources.ru.xaml

```diff
@@ -2,127 +2,127 @@
-    <system:String x:Key="Display_Scaling">Display Scaling (%)</system:String>
-    <system:String x:Key="FPS_Limit">FPS Limit</system:String>
-    <system:String x:Key="ActiveProfile">Active Profile</system:String>
+    <system:String x:Key="Brightness">Яркость</system:String>
+    <system:String x:Key="Volume">Громкость</system:String>
+    <system:String x:Key="TDP">TDP (Вт)</system:String>
+    <system:String x:Key="TDP_Sustained">Стабильный TDP (Вт)</system:String>
+    <system:String x:Key="TDP_Boost">Повышенный TDP (Вт)</system:String>
+    <system:String x:Key="Max_CPU">Максимальная частота CPU (МГц)</system:String>
+    <system:String x:Key="Active_CPU_Cores">Активные ядра CPU</system:String>
+    <system:String x:Key="GPU_Clock">Частота GPU (МГц)</system:String>
+    <system:String x:Key="Display_Resolution">Разрешение дисплея</system:String>
+    <system:String x:Key="Display_Refresh">Частота обновления дисплея</system:String>
+    <system:String x:Key="Display_Scaling">Масштаб дисплея (%)</system:String>
+    <system:String x:Key="FPS_Limit">Ограничение FPS</system:String>
+    <system:String x:Key="ActiveProfile">Активный профиль</system:String>
     
-    <system:String x:Key="SystemControls">SYSTEM CONTROLS</system:String>
+    <system:String x:Key="SystemControls">УПРАВЛЕНИЕ СИСТЕМОЙ</system:String>
     
     <!-- profiles page -->
-    <system:String x:Key="Profiles_Header">Profiles</system:String>
-    <system:String x:Key="Profiles_AddProfile">Add Profile</system:String>
-    <system:String x:Key="Profiles_DeleteProfile">Delete Profile</system:String>
-    <system:String x:Key="Profiles_SaveProfile">Save Profile</system:String>
-    <system:String x:Key="Profiles_HeaderDetails">Profile Details</system:String>
-    <system:String x:Key="Profiles_Name">Profile Name</system:String>
-    <system:String x:Key="Profiles_OnCharger">On Charger Values</system:String>
-    <system:String x:Key="Profiles_OnBattery">On Battery Values</system:String>
-    <system:String x:Key="Profiles_ProfileApplications">Profile Applications</system:String>
-    <system:String x:Key="Profiles_AddApp">Add App</system:String>
-    <system:String x:Key="Profiles_DeleteApp">Delete App</system:String>
+    <system:String x:Key="Profiles_Header">Профили</system:String>
+    <system:String x:Key="Profiles_AddProfile">Добавить профиль</system:String>
+    <system:String x:Key="Profiles_DeleteProfile">Удалить профиль</system:String>
+    <system:String x:Key="Profiles_SaveProfile">Сохранить профиль</system:String>
+    <system:String x:Key="Profiles_HeaderDetails">Детали профиля</system:String>
+    <system:String x:Key="Profiles_Name">Имя профиля</system:String>
+    <system:String x:Key="Profiles_OnCharger">Значения при работе от зарядного устройства</system:String>
+    <system:String x:Key="Profiles_OnBattery">Значения при работе от батареи</system:String>
+    <system:String x:Key="Profiles_ProfileApplications">Связанные приложения</system:String>
```

Comment:
```suggestion
    <system:String x:Key="Profiles_ProfileApplications">Приложения профиля</system:String>
```



Minor: more 1-to-1 translation would be like "Приложения профиля" ("Applications of the Profile"). The current translation "Связанные приложения" means something more like "Linked Applications", so it differs from the original string a bit more than it could

Reply:
Я попытался сохранить смысл. В данном случае идёт перечисление отдельных приложений, запускающихся при активации профиля. Думаю, в таком случае можно сказать, что они связаны